### DISCUSSION TITLE
Use Lage size Images in Snapchat Camera

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#119] [Ideas & Feature Suggestions]
As we all know that the pixels we get from our normal camera isn't achieved by snapchat and the quality is quite dull. Can you try adding HD photos or something like this option for the camera.

Being a developer, I know it's not possible for an app to do so. But just curious if you can try, it will help a lot. Thanks!

**Comment by rodit** [comment]
I can have another look at this although Snapchat's camera code is a complete mess.

**Reply by RevealedSoulEven** [reply]
Yeaaa I came to know over that too

**Reply by RevealedSoulEven** [reply]
https://forum.xda-developers.com/t/magisk-module-snapchat-pixel-mod-better-quality-videos.4325969/

Have a look @rodit

**Reply by rodit** [reply]
It looks like that's for video playback rather than recording.

**Reply by Magna-Graecia** [reply]
@rodit The OP in the XDA thread states that it improves video quality while watching snaps, but the github link he posted as the source states that it improves camera quality in Snapchat. That sounds like it makes images/videos taken with the camera in Snapchat be higher quality, but I haven't used the module personally, so I can't comment on it.

For what it's worth, I don't think the OP in the XDA thread is the creator. I think he's just someone who was posting it as a PSA to other XDA users, and in doing so stated his personal use for it, which is watching snaps in better quality. It might indeed improve camera quality though. Either way, I wouldn't want my props changed just to improve image/video quality on Snapchat. This could cause issues with safetynet down the line. People already use MagiskHide Props Config to basically do the same as what this mod is doing, which is to make your device look like something else so it can pass safetynet while using a custom or uncertified ROM.

I'm on a stock ROM and pass safetynet as soon as I disable systemless Xposed and would like it to stay that way lol.

@RevealedSoulEven By the way, this is off-topic, but do you know any good modules etc like SnapMod but for KIK?

**Reply by RSR1337** [reply]
> https://forum.xda-developers.com/t/magisk-module-snapchat-pixel-mod-better-quality-videos.4325969/
> 
> Have a look @rodit

This will work? 
![Screenshot_20220527-004658_Settings~2](https://user-images.githubusercontent.com/52901789/170578046-45e095d1-8f29-42d8-aeee-8a5ade85ad62.png)

**Reply by rodit** [reply]
I can almost guarantee this won't affect Snapchat
